db/schema.rb

```diff
@@ -448,6 +448,7 @@
   add_index "order_transports", ["scheduled_at"], name: "index_order_transports_on_scheduled_at", using: :btree
 
   create_table "orders", force: :cascade do |t|
+    t.string   "status"
```

Comment:
aren't we removing this field? or later?

Reply:
It's in another branch. I will raise PR for it later separately. :)